src/components/ResourceInstance/Connectivity/CustomDNS.tsx

```
@@ -0,0 +1,393 @@
+            accessQueryParams.serviceEnvironmentKey,
+            accessQueryParams.serviceModelKey,
+            accessQueryParams.productTierKey,
+            resourceKey,
+            accessQueryParams.resourceInstanceId,
+            accessQueryParams.subscriptionId
+          )
+            .then((response) => {
+              const topologyDetails =
+                response.data?.detailedNetworkTopology?.[resourceId];
+              //check for dnsName field in the response, absence means dns removal complete
+              if (!Boolean(topologyDetails?.customDNSEndpoint.dnsName)) {
```

Biome (1.9.4)

[error] Avoid redundant `Boolean` call

It is not necessary to use `Boolean` call when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant `Boolean` call


(lint/complexity/noExtraBooleanCast)